ecl/hql/hqlusage.cpp

```diff
@@ -416,8 +417,9 @@ IPropertyTree * SourceFieldUsage::createReport(bool includeFieldDetail, const IP
             return NULL;
     }
 
-    Owned<IPropertyTree> entry = createPTree(type);
+    Owned<IPropertyTree> entry = createPTree("datasource");
```

Comment:
This is changed so that it can easily be mapped to cassandra tables using existing cassandra "mapping" code.